documentation/site/pages/developers/tutorials/bonsai.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Migrating from Bonsai?
+description: A practical guide for developers transitioning from Bonsai to Boundless
+---
+
+# Migrating from Bonsai?
+
+## What happened to Bonsai?
+
+Bonsai is no longer available for individual developers. Boundless is intended to be a replacement for all your proving needs. If you want to read about the differences between Bonsai and Boundless, please continue on this page. Otherwise, if you have a guest program ready and you are eager to get started, you can follow the [Request a Proof](/developers/tutorials/request) tutorial to create your first proof request.
+
+Adopting Boundless means embracing decentralization; while the initial setup is more complex than Bonsai's  configuration, you gain access to verifiable compute through permissionless, sustainable proving protocol that aligns with Web3's core principles.
+
+## Key Differences
+
+While Bonsai offers simplicity through centralization, Boundless provides:
+
+- *No vendor lock-in*: Decentralized protocol with multiple provers
+- *Censorship resistance*: No single point of failure
+- *Long-term sustainability*: Community-driven protocol evolution
+- *Transparent pricing*: Market-driven proof costs
+
+
+| Aspect | Bonsai | Boundless |
```

Comment:
I feel the key difference is that Boundless delivers proofs onchain. I guess thats kind of called out implicitly in Proof Verification, but could be made more explicit by just having Proof Delivery as a row, with onchain/offchian for options